packages/toolbox/src/chainlinkConversionPathTools.ts

```diff
@@ -154,7 +154,7 @@ const addSupportedCurrency = (
   record[type] = record[type].sort();
 };
 
-export const listAggregators = async (options?: IOptions): Promise<void> => {
+export const listAggregators = async (options?: IOptions): Promise<any> => {
```

Comment:
Ideally, can you type the return promise ?

Reply:
I didn't even end up using this, so I'm returning it to void